lib/translator/inquiry.cc

```diff
@@ -158,6 +231,69 @@ void TranslateUnitSerialNumberVpd(
   memcpy(&buffer[sizeof(result)], &product_serial_number, result.page_length);
 }
 
+// Returns Extended Inquiry Data Page. Refer to section 6.1.5
+// https://nvmexpress.org/wp-content/uploads/NVM_Express_-_SCSI_Translation_Reference-1_5_20150624_Gold.pdf
+void TranslateExtendedInquiryDataVpd(
+    const nvme_defs::IdentifyNamespace &identify_namespace_data,
+    const nvme_defs::IdentifyControllerData &identify_controller_data,
+    absl::Span<uint8_t> buffer) {
+  uint8_t dps_result =
+      identify_namespace_data.dps.md_start || identify_namespace_data.dps.pit;
+  uint8_t spt = GetSptValueFromDPC(identify_namespace_data);
+
+  scsi_defs::ExtendedInquiryDataVpd extended_inquiry_data = {
+
```

Comment:
nit: mentioned in other PRs, but just observing again that all these newlines create a logical disconnect when reading this structure for me

Reply:
i feel removing the newlines makes the code pretty cluttered and hard to read. My intention was to add one line between each field because of the presence of comments above each field. This divides it into a few "comment/field initializer blocks and makes it easier for the reader to read.